packages/nextra-theme-docs/src/index.tsx

```diff
@@ -119,27 +119,19 @@ const Body: React.FC<BodyProps> = ({
           </main>
         </article>
       )}
-    </React.Fragment>
+    </>
   )
 }
-interface LayoutProps {
-  filename: string
-  pageMap: PageMapItem[]
-  meta: Meta
-  titleText: string | null
-  headings?: Heading[]
-  timestamp?: number
-}
 
-const Content: React.FC<LayoutProps> = ({
+const Content = ({
   filename,
   pageMap,
   meta,
   titleText,
   headings,
   timestamp,
   children
-}) => {
+}: PageOpts & { children: ReactNode }): ReactElement => {
```

Comment:
don't need to specify props since we'll receive all PageOpts anyway

https://github.com/shuding/nextra/blob/153aee61f7463752608f021a7ff9e730fcef2856/packages/nextra-theme-docs/src/index.tsx#L243
https://github.com/shuding/nextra/blob/153aee61f7463752608f021a7ff9e730fcef2856/packages/nextra-theme-docs/src/index.tsx#L261

Reply:
could use PropsWithChildren<PageOpts> here

Reply:
PropsWithChildren has optional children, and like this his is required